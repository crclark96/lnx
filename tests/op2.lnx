*ah;
**dx;
/cl;
//bx;
-ax;
^rbx;
