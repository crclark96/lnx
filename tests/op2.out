imul ah
mul dx
idiv cl
div bx
neg ax
